js/settings.js

```diff
@@ -91,6 +91,15 @@ module.exports = function(opts) {
     if (!arguments.length) return settings;
 
     settings = data.grouped.settings;
+    var firstBgTarget = settings[settings.length - 1].bgTarget[0];
```

Comment:
So if you look at [the data model for settings](http://developer.tidepool.io/data-model/v1/settings/), there are four different ways of storing `bgTarget`. Until now, tideline only needed to display one because we only had one source of pump data. Eventually we should account for all of these in a more flexible way, but to be honest we developed the four different ways of storing `bgTarget` in the data model somewhat in the abstract, and until we have confirmation that there actually *are* pumps that store this setting in each of these different ways, it feels silly to me to make tideline capable of displaying all of them. (Also some of the others may require much more work to make them fit into the design.)

Is that what you're asking?